ui/scss/component/_file-render.scss

```diff
@@ -580,6 +504,42 @@ $control-bar-icon-size: 0.8rem;
     &.skip {
       background-color: rgba(0, 0, 0, 0.5);
     }
+
+    // Override the original's 'display: block' to avoid the big play button
+    // from being squished to the side:
+    position: absolute;
+  }
+}
+
+video::-internal-media-controls-overlay-cast-button {
+  // Push the cast button above vjs-touch-overlay:
+  z-index: 3;
+
+  // Move it to the upper-right since it will clash with "tap to unmute":
+  left: unset;
+  right: 8px;
+}
+
+.video-js.video-js.vjs-user-inactive {
```

Comment:
These were removed in 1766b418 ... maybe missed in the merge? (I assume you've been merging from master)

Reply:
Yes I am. Not quite sure how this happend, but it's fixed.